$OpenBSD: patch-configure_ac,v 1.2 2011/09/25 15:26:04 ajacoutot Exp $

- Fix path to soundcard.h
- Use /var/db instead of /var/lib

--- configure.ac.orig	Thu Jun 23 22:56:05 2011
+++ configure.ac	Sun Sep 25 17:21:24 2011
@@ -104,7 +104,7 @@ if test "x$M4" = xno ; then
 fi
 
 dnl Compiler flags
-CC_CHECK_CFLAGS_APPEND([-Wall -W -Wextra -pipe -Wno-long-long -Winline -Wvla -Wno-overlength-strings -Wunsafe-loop-optimizations -Wundef -Wformat=2 -Wlogical-op -Wsign-compare -Wformat-security -Wmissing-include-dirs -Wformat-nonliteral -Wold-style-definition -Wpointer-arith -Winit-self -Wdeclaration-after-statement -Wfloat-equal -Wmissing-prototypes -Wstrict-prototypes -Wredundant-decls -Wmissing-declarations -Wmissing-noreturn -Wshadow -Wendif-labels -Wcast-align -Wstrict-aliasing=2 -Wwrite-strings -Wno-unused-parameter -ffast-math -Wp,-D_FORTIFY_SOURCE=2 -fno-common -fdiagnostics-show-option])
+CC_CHECK_CFLAGS_APPEND([-Wall -W -Wextra -pipe -Wno-long-long -Winline -Wvla -Wno-overlength-strings -Wunsafe-loop-optimizations -Wundef -Wformat=2 -Wlogical-op -Wsign-compare -Wformat-security -Wmissing-include-dirs -Wformat-nonliteral -Wold-style-definition -Wpointer-arith -Winit-self -Wdeclaration-after-statement -Wfloat-equal -Wmissing-prototypes -Wstrict-prototypes -Wmissing-declarations -Wmissing-noreturn -Wshadow -Wendif-labels -Wcast-align -Wstrict-aliasing=2 -Wwrite-strings -Wno-unused-parameter -ffast-math -Wp,-D_FORTIFY_SOURCE=2 -fno-common -fdiagnostics-show-option])
 
 dnl Linker flags.
 dnl Check whether the linker supports the -version-script option.
@@ -750,7 +750,7 @@ AC_ARG_ENABLE([oss-wrapper],
         [oss_wrapper=auto])
 
 if test "x${oss_output}" != xno || test "x${oss_wrapper}" != "xno"; then
-    AC_CHECK_HEADERS([sys/soundcard.h],
+    AC_CHECK_HEADERS([soundcard.h],
         [
             if test "x${oss_output}" != "xno"; then
                 AC_DEFINE([HAVE_OSS_OUTPUT], 1, [Have OSS output?])
@@ -1374,9 +1374,9 @@ fi
 #### PulseAudio system runtime dir ####
 PA_SYSTEM_RUNTIME_PATH="${localstatedir}/run/pulse"
 AC_SUBST(PA_SYSTEM_RUNTIME_PATH)
-PA_SYSTEM_CONFIG_PATH="${localstatedir}/lib/pulse"
+PA_SYSTEM_CONFIG_PATH="${localstatedir}/db/pulse"
 AC_SUBST(PA_SYSTEM_CONFIG_PATH)
-PA_SYSTEM_STATE_PATH="${localstatedir}/lib/pulse"
+PA_SYSTEM_STATE_PATH="${localstatedir}/db/pulse"
 AC_SUBST(PA_SYSTEM_STATE_PATH)
 
 ###################################
